src/settings/components/TemplateEditor/EditorToolbar/EditorToolbar.js

```diff
@@ -4,34 +4,34 @@ const EditorToolbar = () => {
   return (
     <div id="toolbar">
       <span className="ql-formats">
-        <button type="button" className="ql-bold" />
-        <button type="button" className="ql-italic" />
-        <button type="button" className="ql-underline" />
+        <button type="button" aria-label="bold" className="ql-bold" />
+        <button type="button" aria-label="italic" className="ql-italic" />
+        <button type="button" aria-label="underline" className="ql-underline" />
       </span>
       <span className="ql-formats">
-        <button type="button" className="ql-list" value="ordered" />
-        <button type="button" className="ql-list" value="bullet" />
+        <button type="button" aria-label="ordered list" className="ql-list" value="ordered" />
+        <button type="button" aria-label="bugllet list" className="ql-list" value="bullet" />
       </span>
       <span className="ql-formats">
-        <button type="button" className="ql-indent" value="-1" />
-        <button type="button" data-test-increase-indent className="ql-indent" value="+1" />
+        <button type="button" aria-label="decrease indent" className="ql-indent" value="-1" />
+        <button type="button" aria-label="increase indent" data-test-increase-indent className="ql-indent" value="+1" />
       </span>
       <span className="ql-formats">
         <select className="ql-size">
-          <option value="10px" />
-          <option defaultValue />
-          <option value="18px" />
-          <option value="32px" />
+          <option aria-label="10 pixels" value="10px" />
+          <option aria-label="default size" defaultValue />
+          <option aria-label="18 pixels" value="18px" />
+          <option aria-label="32 pixels" value="32px" />
         </select>
       </span>
       <span className="ql-formats">
-        <button type="button" className="ql-align" value="" />
-        <button type="button" className="ql-align" value="center" />
-        <button type="button" className="ql-align" value="right" />
-        <button type="button" className="ql-align" value="justify" />
+        <button type="button" aria-label="no align" className="ql-align" value="" />
+        <button type="button" aria-label="center" className="ql-align" value="center" />
+        <button type="button" aria-label="right" className="ql-align" value="right" />
+        <button type="button" aria-label="justify" className="ql-align" value="justify" />
       </span>
       <span className="ql-formats">
-        <button type="button" className="ql-link" />
+        <button type="button" aria-label="link" className="ql-link" />
       </span>
```

Comment:
I filed [UICIRC-421](https://issues.folio.org/browse/UICIRC-421) to translate these `aria-label` values. There is no need to complete those translations in the scope of this story, but any time you find your self inserting English text into a value, you should think about i18n.